webpack.tasks/js.loader.js

```diff
@@ -17,17 +17,8 @@ const jsLoader = () => {
         loader: 'babel-loader',
         options: {
             cacheDirectory: true,
-            presets: [
-                [
-                    'env',
-                    {
-                        targets: {
-                            browsers: ['ie 11']
-                        }
-                    }
-                ]
-            ],
-            plugins: [require('babel-plugin-transform-object-rest-spread'), require('babel-plugin-transform-runtime')],
+            presets: ['env'],
```

Comment:
THeese 2 lines eliminates babel-polyfill (core-js) from app-vendor. Its uncessary to include only polyfills for IE11, if you are including whole polyfill in vendors anyway. If needed only some polyfills, you should remove babel-polyfill in vendors, and leave this here.